pkgs/by-name/so/soundalike/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  chromaprint,
+  makeWrapper,
+}:
+
+buildGoModule rec {
+  pname = "soundalike";
+  version = "0.1.2";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "derat";
+    repo = "soundalike";
+    rev = "v${version}";
+    hash = "sha256-mpYUVTj3Zll6kNuK5Mdzv1R7k5FZy6XFghhzmAPPVM8=";
+  };
+
+  vendorHash = "sha256-7hRezOBcjB2wsx/SwV519wg3Azh+0kHMcAoc9aYPM3A=";
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ chromaprint ];
+
+  doCheck = true;
+  checkFlags = [
+    "-skip=TestMain_Scan|TestMain_Compare"
```

Comment:
It can be helpful to add a comment on why these tests need to be disabled, for example because they require network connectivity or have some dependency that is not (yet) packaged.

Reply:
Why not provide in PATH?

This might do it:

```
  preCheck = ''export PATH="$GOPATH/bin:$PATH"'';
```